aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Avatar, Box, Text } from 'grommet';
+
+export const AvatarSizePreviewExample = () => (
+  <Box align="center" height="small" direction="row" gap="xlarge">
+    <AvatarContainer
```

Comment:
Alignment looks kind of strange here -- maybe left-align avatar and text?

Reply:
alignment with the text under the Avatar ?